tests/python/pytorch/graphbolt/impl/test_minibatch.py

```diff
@@ -0,0 +1,97 @@
+import dgl
+import dgl.graphbolt as gb
+import torch
+
+
+def test_to_dgl_blocks_hetero():
+    relation = "A:relation:B"
+    node_pairs = {relation: (torch.tensor([0, 1, 2]), torch.tensor([0, 4, 5]))}
+    reverse_column_node_ids = {"B": torch.tensor([10, 11, 12, 13, 14, 16])}
+    reverse_row_node_ids = {
+        "A": torch.tensor([5, 9, 7]),
+        "B": torch.tensor([10, 11, 12, 13, 14, 16]),
+    }
+    reverse_edge_ids = {relation: torch.tensor([19, 20, 21])}
+    node_features = {
+        ("A", "x"): torch.randint(0, 10, (3,)),
+    }
+    edge_features = {(relation, "x"): torch.randint(0, 10, (3,))}
+    subgraph = gb.SampledSubgraphImpl(
+        node_pairs=node_pairs,
+        reverse_column_node_ids=reverse_column_node_ids,
+        reverse_row_node_ids=reverse_row_node_ids,
+        reverse_edge_ids=reverse_edge_ids,
+    )
+    blocks = gb.MiniBatch(
+        sampled_subgraphs=[subgraph, subgraph],
```

Comment:
Let's spend slightly more effort here to make the data correct. The 2 subgraph should be different, and it is better to show that the dst of the first layer is the src of the second layer.